### PR TITLE
Pract.06: Add 06-07 | Stefka Lyaskalieva

### PR DESCRIPTION
**Pull request by stefka04**
Stefka Lyaskalieva
2 4MI0600158

### CODE REVIEW

File: SI/Sem.06/Pract.06/Task6.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+using namespace std;
+const int SIZE = 7;
+void inputArray(int* arr)
+{
+	for (int i = 0; i < SIZE; i++) 
+	{
+	cin >> arr[i];
+	}
+}
+int checkUniqueNumber(int* arr)
```
Comment (Justsvetoslavov) [inline SI/Sem.06/Pract.06/Task6.cpp:11]: int checkUniqueNumber(const int* arr, int size)

File: SI/Sem.06/Pract.06/Task07.cpp [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	for (int i = 0; i < SIZE; i++)
+	{
+		cin >> arr[i];
+	}
+}
+int powerOfTwo(int exp)
+{
+	return (1 << exp);
+}
+  void getSubNumber(int* arr, int mask)
```
Comment (Justsvetoslavov) [inline SI/Sem.06/Pract.06/Task07.cpp:15]: void getSubNumber(const int* arr, int size, int mask)

**Review by Justsvetoslavov — COMMENTED** [review]
Small changes, overall KUDOS👏

**Review by Justsvetoslavov — APPROVED** [review]

**Comment by Justsvetoslavov** [comment]
@dejansvetoslavov @GeorgiTerziev02 Requesting 2nd review

**Review by GeorgiTerziev02 — APPROVED** [review]
It is better to pass the size of the array as function parameter

**Comment by Justsvetoslavov** [comment]
@stefka04 Can you address the PR comments ?
I want to merge the solutions : )

**Pull request merged**